policy_engine: rename inbound/outbound to new phase names (#955)
ngrok/ngrok-docs: traffic-policy/actions/remove-headers/behavior.mdx

## Behavior

> When executed during the `on_http_request` phase, this action will remove headers on an incoming http request before reaching the upstream server with
> the configured headers. When executed during the `on_http_response`, the configured headers are removed to the response from the upstream server.

Reviewer: ```suggestion
When executed during the `on_http_request` phase, this action will remove the specified headers from incoming http requests before reaching the upstream server. When executed during the `on_http_response`, the configured headers are removed from http responses returned from the upstream server before reaching the client.
```
